endpointmanager/pkg/capabilityparser/factory.go

```diff
@@ -101,6 +103,48 @@ func (resp *Response) Equal(resp2 SMARTResponse) bool {
 	return true
 }
 
+// EqualIgnore checks if the smart response body is equal to the given smart response body while ignoring certain fields that may differ.
```

Comment:
This maybe doesn't need to be addressed in this PR, but is there a reason why the smart response functions don't have their own separate file @mayerm94? It kind of seems like at this point it should be separated out instead of mixed in with the Capability Statement stuff, but maybe I'm missing something

Reply:
i agree, I think it would make sense (in task to add to the backlog) to separate out the SMARTResponse type to have it's own factory. This would make it easier to handle different versions of SMART Responses should they come about in the future